fulp_modules/batong/code/batongcode.dm

```diff
@@ -0,0 +1,11 @@
+
+
+/obj/item/toy/plush/batong
+	name = "\improper batong"
```

Comment:
Why is this \improper?

Reply:
i believe that was my fault, i probably gave him the wrong explanation reguarding the use of improper in the name.